src/SumoCoders/FrameworkCoreBundle/Resources/assets/js/Framework/Scrolling.js

```diff
@@ -1,4 +1,8 @@
 export class Scrolling {
+  constructor () {
+    this.initEventListeners()
```

Comment:
Can't the initEventListeners code not come in the constructor?

Reply:
Wouldn't that make the constructor a bit messy?